mmdet3d/models/middle_encoders/pillar_scatter.py

```diff
@@ -50,7 +50,7 @@ def forward_single(self, voxel_features, coors):
             dtype=voxel_features.dtype,
             device=voxel_features.device)
 
-        indices = coors[:, 1] * self.nx + coors[:, 2]
+        indices = coors[:, 2] * self.nx + coors[:, 3]
```

Comment:
Hi, you can `return canvas`  in forward_single directly to make it consistent with forward_batch.